core/icmp-jna/src/main/java/org/opennms/jicmp/standalone/PingReply.java

```diff
@@ -33,6 +33,9 @@
 /**
  * PingReply
  *
+ * @deprecated This interface has a lot of overlap with {@link EchoPacket}...
+ * one of them needs to be moved into an API project.
+ * 
  * @author brozow
  */
 interface PingReply {
```

Comment:
Please also add `@Deprecated`. The Annotation is interpreted by the IDE, whereas the javadoc bit is for documentation purposes.